src/OSPSuite.TeXReporting/OSPSuite.TeXReporting.csproj

```diff
@@ -22,7 +22,7 @@
     <WarningLevel>4</WarningLevel>
     <DocumentationFile>bin\Debug\OSPSuite.TeXReporting.xml</DocumentationFile>
     <NoWarn>1591</NoWarn>
-    <PlatformTarget>x86</PlatformTarget>
+    <PlatformTarget>AnyCPU</PlatformTarget>
```

Comment:
This was not set properly. WE found that out as part of our work on x64 bit. Might as well fix it